tests/lib/rules/no-classic-classes.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-classic-classes');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE_NO_CLASSIC_CLASSES: ERROR_MESSAGE } = rule;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+});
+
+ruleTester.run('no-classic-classes', rule, {
+  valid: [
+    `
+      import Component from '@ember/component';
```

Comment:
It felt redundant to test against imported classes from a bunch of different Ember modules, but I can bring those back if we want to also test against Controller, Route, etc.